src/test/java/com/github/underscore/CollectionsTest.java

```diff
@@ -114,23 +114,23 @@ public void apply(final Map.Entry<String, Integer> item) {
     @SuppressWarnings("unchecked")
     public void forEachRight() {
         final List<Integer> result = new ArrayList<Integer>();
-        $.forEachRight(asList(1, 2, 3), new Block<Integer>() {
-            public void apply(Integer item) {
+        $.forEachRight(asList(1, 2, 3), new Consumer<Integer>() {
+            public void accept(Integer item) {
                 result.add(item);
             }
         });
         assertEquals("[3, 2, 1]", result.toString());
         final List<Integer> result2 = new ArrayList<Integer>();
-        new $(asList(1, 2, 3)).forEachRight(new Block<Integer>() {
-            public void apply(Integer item) {
+        new $(asList(1, 2, 3)).forEachRight(new Consumer<Integer>() {
+            public void accept(Integer item) {
                 result2.add(item);
             }
         });
         assertEquals("[3, 2, 1]", result2.toString());
         final List<Map.Entry<String, Integer>> resultChain = new ArrayList<Map.Entry<String, Integer>>();
         $.chain((new LinkedHashMap<String, Integer>() { { put("a", 1); put("b", 2); put("c", 3); } }).entrySet())
-            .forEachRight(new Block<Map.Entry<String, Integer>>() {
-                public void apply(final Map.Entry<String, Integer> item) {
+            .forEachRight(new Consumer<Map.Entry<String, Integer>>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.forEachRight\(Consumer\) ignored, is this OK in com\.github\.underscore\.CollectionsTest\.forEachRight\(\)